flaml/tune/tune.py

```diff
@@ -300,6 +356,18 @@ def easy_objective(config):
         max_failure: int | the maximal consecutive number of failures to sample
             a trial before the tuning is terminated.
         use_ray: A boolean of whether to use ray as the backend.
+        lexico_objectives: A dictionary with four elements.
+        It specifics the information used for multiple objectives optimization with lexicographic preference.
+        e.g.,```lexico_objectives = {"metrics":["error_rate","pred_time"], "modes":["min","min"],
+        "tolerances":{"error_rate":0.01,"pred_time":0.0}, "targets":{"error_rate":0.0,"pred_time":0.0}}```
+
+        Either "metrics" or "modes" is a list of str.
+        It represents the optimization objectives, the objective as minimization or maximization respectively.
+        Both "metrics" and "modes" are ordered by priorities from high to low.
+        "tolerances" is a dictionary to specify the optimality tolerance of each objective.
+        "targets" is a dictionary to specify the optimization targets for each objective.
+        If providing lexico_objectives, the arguments metric, mode, and search_alg will be invalid.
+

```

Comment:
```suggestion
            It specifics the information used for multiple objectives optimization with lexicographic preference.
            e.g.,
            ```python
            lexico_objectives = {"metrics":["error_rate","pred_time"], "modes":["min","min"],
            "tolerances":{"error_rate":0.01,"pred_time":0.0}, "targets":{"error_rate":0.0,"pred_time":0.0}}
            ```
            Either "metrics" or "modes" is a list of str.
            It represents the optimization objectives, the objective as minimization or maximization respectively.
            Both "metrics" and "modes" are ordered by priorities from high to low.
            "tolerances" is a dictionary to specify the optimality tolerance of each objective.
            "targets" is a dictionary to specify the optimization targets for each objective.
            If providing lexico_objectives, the arguments metric, mode, and search_alg will be invalid.
```
Please double check how it looks in the rendered page.